.github/workflows/ci.yml

```diff
@@ -126,7 +126,7 @@ jobs:
     - name: Run the tests with x509-parser enabled
       run: cargo test --verbose --features x509-parser --all-targets
     - name: Run the tests with no default features enabled
-      run: cargo test --verbose --no-default-features --all-targets
+      run: cargo test --verbose --all-targets
```

Comment:
The `name` and `run` have fallen out of sync here.

Reply:
`cargo test` is already another step. Probably better to have a proper test on CI with `aws_lc_rs`. I am going to change accordingly